### DISCUSSION TITLE
May I ask for the sample format of the input data?    /(ㄒoㄒ)/~~

### DISCUSSION THREAD

**Discussion by bobomie** [nzahasan/Tank-Model#20] [General]
I have some questions on  project file .  
firstly,  in the  file of " test >- test_data >- channel_routing_muskingum.json ",  may i know how to get  double outflow ?  (because it is calculated by the code ,why it can be the input data?) 
secondly, i have no idea about the basin file , Is there any possible that i can get more detailed information，
Thank u soooo munch！ Hope all is well.

**Comment by nzahasan** [comment]
1. test data are for unit-testing basically to check if the developed functions are correct. 
2. This model does not have any geo-processing functionality (delineate basin, river network etc.) for this we use [HEC-HMS](https://www.hec.usace.army.mil/software/hec-hms/) to delineate basin structure and then convert the HMS delineated basin to and convert to Tank Model supported basin structure. In HMS basin definition can be found in the project folder in a file with `.hms` extension.

I am attaching a sample project for your study.

[brahmaputra-tank-test.zip](https://github.com/user-attachments/files/17414776/brahmaputra-tank-test.zip)

**Reply by bobomie** [reply]
Got it !  And I will keep on studying. Thank you sincerely!  *^____^*
